scenarios/appsec/test_client_ip.py

```diff
@@ -11,7 +11,7 @@
 
 
 @released(dotnet="?", golang="?", java="?")
-@released(nodejs="?", php="?", python="?", ruby="?")
+@released(nodejs="?", php_appsec="0.4.4", python="?", ruby="?")
```

Comment:
I think in this PR we are not testing with appsec:0.4.4
I guess it is the same case as: https://github.com/DataDog/system-tests/pull/600#discussion_r1006915869

Reply:
I tested it in local and it worked. @Anilm3 would you mind to give it a second run to confirm it's all working?

Reply:
@robertomonteromiguel is right, this particular test verifies that the IP isn't added when appsec is disabled, we don't have this change merged yet.